src/capitalsFirst.js

```diff
@@ -18,7 +18,19 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let initialArray = str.split(' ');
+  let uppercaseLetterFirst = [];
+  let lowercaseLetterFirst = [];
+
+  for (let i = 0; i < initialArray.length; i++) {
+    if (initialArray[i][0] >= 'A' && initialArray[i][0] <= 'Z') {
```

Comment:
do not hardcode strings inside code move it to constants outside of function and name with UPPER_CASE
ex.: const FIRST_LOWERCASE_LETTER = 'a';
Also, you can move initialArray[i][0] >= 'A' && initialArray[i][0] <= 'Z' to isUppercaseLetter helper function